src/Services/Service.js

```diff
@@ -7,7 +7,8 @@ export var Service = L.Evented.extend({
 
   options: {
     proxy: false,
-    useCors: cors
+    useCors: cors,
+    timeout: 5000
```

Comment:
Lets keep the timeout option at the `XMLHTTPRequest` default of `0`. Otherwise this feels like a breaking change.